noisyatom/config/settings.py

```diff
@@ -38,25 +38,25 @@
     # Database
     # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
     # Setup the database as a developer machine
-    # DATABASES = {
-    #     'default': {
-    #         'ENGINE': 'django.db.backends.postgresql',
-    #         'NAME': 'ffw',
-    #         'USER': 'ffw_user',
-    #         'PASSWORD': '1234',
-    #         'HOST': '127.0.0.1',
-    #         'PORT': '5432',
-    #     }
-    # }
+    DATABASES = {
+         'default': {
+             'ENGINE': 'django.db.backends.postgresql',
+             'NAME': 'ffw',
+             'USER': 'ffw_user',
+             'PASSWORD': '1234',
+             'HOST': '127.0.0.1',
+             'PORT': '5432',
+         }
+     }
 
     # Making our dev machines use sqlite for now. It makes running up a dev instance from scratch quicker for demo
     # purposes.
-    DATABASES = {
```

Comment:
Can you add in the sql-lite back for developers :-) 
It's fine for you to leave your code in here as it does nothing.

Reply:
No problem. Changed back to use sql-lite db.